include/meta/meta.hpp

```diff
@@ -3162,12 +3173,15 @@ namespace meta
         /// \f$ O(log(N)) \f$.
         /// \ingroup integral
         template <typename T, T N>
-        using make_integer_sequence =
+        using make_integer_sequence = _t<
             _t<detail::coerce_indices_<T, 0, make_index_sequence<static_cast<std::size_t>(N)>>>;
```

Comment:
I am interested about the additional `_t<` which does that not compile (@CaseyCarter)? Is somebody kind to explain what this is meant for? =) What is the advanced technique here? :-)
BR